orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -29,6 +31,7 @@
 CONFIG_STREAM_NAME = 'configs'
 SHARED_MCONFIG = 'shared_mconfig'
 MAGMAD = 'magmad'
+VersionInfo = namedtuple('VersionInfo', 'agw_version orc8r_version')
```

Comment:
```suggestion
VersionInfo = NamedTuple('VersionInfo', 
              [('agw_version', str), ('orc8r_version', str)],
)
```
prefer `typing.NamedTuple` usage